### ISSUE TITLE
Notifications for publish builds

### ISSUE THREAD

**Issue by bluekeyes** [palantir/gerrit-ci#20]
@mnazbro pointed out that while Gerrit CI comments on CRs when verification builds start and finish, it doesn't comment on anything for publish builds. While it's not obvious to me where it should comment, there's currently no way to know when a publish build fails (or succeeds) other than checking the Jenkins job or looking in the artifact store for your new artifact.

Maybe we could support email alerts for published builds? Comment on the CR for the tip commit on the branch being published? Add a new tab for build statuses?

**Comment by newren** [comment]
Yeah, we discussed this for a while and didn't come up with a good answer.  I think someone looked at stashbot and said it had the same issue, and we didn't push any further.

Also, note that there may not be a CR for the tip commit on the branch being published.  (In fact, there almost never will be since CR's are brought into a branch via a merge; that combined with the possibility for direct pushes that bypass code review just means I think CRs are the wrong place to put the information.)  Email alerts or a new tab for build statuses both sound like good potential options, though.

**Comment by mnazbro** [comment]
I would be satisfied with a new tab for build statuses + optional email alerts. That would solve our current problem of unwatched build failures.
